### ISSUE TITLE
Version 55 removed from public ECR on AWS

### ISSUE THREAD

**Issue by MatejBalantic** [DataDog/datadog-lambda-extension#200]
_Update: I fixed a typo below indicating that we've been using version `55` for weeks, not `latest`._

We've been using `public.ecr.aws/datadog/lambda-extension:55` for weeks in our production builds. Today, all of the sudden, our build started failing due to this error:
```
#4 CANCELED
#3 [internal] load metadata for public.ecr.aws/datadog/lambda-extension:55
#3 ERROR: public.ecr.aws/datadog/lambda-extension:55: not found
```

Seems like version 55 disappeared from the public AWS ECR? I can confirm this by looking into the gallery:
https://gallery.ecr.aws/datadog/lambda-extension

Version 55 is nowhere to be found.

We've worked around this by referring to `latest`. 

Versions really should not be disappearing from the registry once they are released.

**Comment by alexgallotta** [comment]
We are investigating the issue. Meanwhile, if it's a possibility, consider pulling the images from dockerhub https://hub.docker.com/r/datadog/lambda-extension/

**Comment by alexgallotta** [comment]
The issue has been resolved, v55 is back online in AWS ECR

**Comment by alexgallotta** [comment]
@MatejBalantic just to clarify: can you confirm that you were using v55 for a while, and you didn't upgrade from v53?

**Comment by MatejBalantic** [comment]
Actually, no, you are right. I reviewed our pipeline history again now, and it looks like we only thought that we were using version 55, but in reality, we were still on version 53. Something unrelated in our source code triggered the Docker image to rebuild today, and it failed, leading us to think that version 55 was removed. But now I understand that it just might have never been there.

Sorry about leading you in the wrong direction. It's still great that version 55 is now available on AWS; thank you!
